.github/workflows/testLatestPublish.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    tags:
+      - '*'
+  # Scheduled jobs will only run on the default branch
+  schedule:
+    - cron: '* */5 * * *' # Run every 5 hours
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      # Sleep when triggered by new tag to wait for npm publish
+      - name: Sleep For 3 Mins
+        run: sleep 180s
+        shell: bash
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+      - name: Install Deps
+        run: npm install
+      - name: Test Latest CLI Release
+        run: |
+          npm run test-cli --cliNPMVersion="5.0.2"
```

Comment:
Purposefully using a bad CLI release on the first pass so we can watch this fail

Reply:
Don't forget to update this to `latest` before merging!